source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -1519,7 +1528,7 @@ AdaptiveNamespaceStart
         AdaptiveSubmitAction();
 
         Windows.Data.Json.JsonValue DataJson;
-        Boolean IgnoreInputValidation;
```

Comment:
was this ever wired up, or just in the object model?

Reply:
Just object model, it never did anything.

---
In reply to: [537961179](https://github.com/microsoft/AdaptiveCards/pull/5141#discussion_r537961179) [](ancestors = 537961179)